packages/material-ui/src/Select/Select.js

```diff
@@ -101,6 +101,7 @@ Select.propTypes = {
   classes: PropTypes.object.isRequired,
   /**
    * If `true`, the selected item is displayed even if its value is empty.
+   * If you want to show a meaningful display, the `renderValue` property must be used to check for the case of an empty selected value and provide a custom render value.
```

Comment:
I think it looks better if we add a blank line under the first sentence. You can do that by adding three new lines in the code.

```suggestion
   *
   *
   *
   * If you want to show a meaningful display, the `renderValue` property must be used to check for the case of an empty selected value and provide a custom render value.
```

What do you think?

Reply:
cc @oliviertassinari

Reply:
Attached photos w/o and w/ the blank line 

w/o blank line:
<img width="873" alt="Screen Shot 2019-06-25 at 5 42 54 PM" src="https://user-images.githubusercontent.com/7755586/60142941-e5db5a00-9770-11e9-8893-6ad73c462361.png">

w/ blank line:
<img width="900" alt="Screen Shot 2019-06-25 at 5 43 53 PM" src="https://user-images.githubusercontent.com/7755586/60142945-e8d64a80-9770-11e9-8355-f4aff3916102.png">

Reply:
Do we have a better way to add these blank line? I believe it's not the only case where we face the problem. What about sticking to?
```js
   * If `true`, the selected item is displayed even if its value is empty.
   *
   * If you want to show a meaningful display, the `renderValue` property must be used to check for the case of an empty selected value and provide a custom render value.
   * You can only use it when the `native` property is `false` (default).
```

There a 35 other occurrences of this blank line approach in the codebase. Maybe we can handle it at the docs generator level.

Reply:
Yeah, it would be better to handle it at the generator layer. I just suggested something that had the correct output

Reply:
btw, what tool are you guys using to generate the docs?

Reply:
@joshwooding Oh yeah, for the blank line, I'm all 👌 for the proposal.

@bigtone1284 We are using react-docgen.